python/tests/spark_test_case.py

```diff
@@ -0,0 +1,69 @@
+'''
+    A parent class for all Flint unit tests
+'''
+#  Copyright 2015-2017 TWO SIGMA OPEN SOURCE, LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+import os
+import sys
+import unittest
+import collections
+from pyspark import SparkContext, SparkConf
```

Comment:
This import should probably go inside setup. Because internally we don't have pyspark until our internal library finishes initializing python path